internal/database/security_event_logs_test.go

```diff
@@ -69,4 +71,70 @@ func TestSecurityEventLogs_ValidInfo(t *testing.T) {
 			assert.Equal(t, tc.err, got)
 		})
 	}
+
+	logs := exportLogs()
+	auditLogs := filterAudit(logs)
+	assert.Equal(t, 3, len(auditLogs))
+	for _, auditLog := range auditLogs {
+		assertAuditField(t, auditLog.Fields["audit"].(map[string]any))
+		assertEventField(t, auditLog.Fields["event"].(map[string]any))
+	}
+}
+
+func filterAudit(logs []logtest.CapturedLog) []logtest.CapturedLog {
+	var filtered []logtest.CapturedLog
+	for _, log := range logs {
+		if log.Fields["audit"] != nil {
+			filtered = append(filtered, log)
+		}
+	}
+	return filtered
+}
+
+func assertAuditField(t *testing.T, field map[string]any) {
```

Comment:
Use `t.Helper` for helpers like this so you get correct line numbers when assertions fail.

```suggestion
func assertAuditField(t *testing.T, field map[string]any) {
    t.Helper()
```

Reply:
Watch out though: that's not a `<tab>` in here because GitHub's web interface doesn't allow me to add tabs... So you need to fix it locally.